### PR TITLE
Article about Increasing Email Deliverability

### PR DESCRIPTION
**Pull request by ThomasLehnerSpryker**
## PR Description

TBD

## Checklist
- [ x] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/ca/dev/email-service/email-deliverability.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+If the server and domain has been reported before for spamming or distributing malicious content, it can be that the message that it wants to transmit will either be outright rejected or flagged as spam.
+There are several domain reputation checkers available that you can use to assess the reputation of your email domain.
+To improve your domain's reputation and increase the deliverability of your emails, there are several things you should do:
+
+#### Debug and troubleshoot carefully
+Your Spryker PaaS non-production environments are - by default - using a sandboxed Simple Email Service Account. This means that you can only send emails to [validated recipients](https://docs.spryker.com/docs/ca/dev/email-service/verify-email-addresses.html).
+This protects you from mistakenly sending faulty emails to many recipients that might report this behavior and damage the reputation of your sender's domain.
+Only request the SES Sandbox to be disabled if you are confident that your email functionality works as expected
```
Comment (profuel) [inline docs/ca/dev/email-service/email-deliverability.md:21]:
```suggestion
Only request the SES Sandbox to be disabled if you are confident that your email functionality works as expected.
```

File: docs/ca/dev/email-service/email-deliverability.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+### Reputation
+In an effort to reduce spam reaching their customers, many email service providers work with reputation databases. If a remote email server tries to connect to and deliver an email to the receiving email server, this server can check the reputation of the sending server and sender domain.
+If the server and domain has been reported before for spamming or distributing malicious content, it can be that the message that it wants to transmit will either be outright rejected or flagged as spam.
+There are several domain reputation checkers available that you can use to assess the reputation of your email domain.
+To improve your domain's reputation and increase the deliverability of your emails, there are several things you should do:
+
+#### Debug and troubleshoot carefully
+Your Spryker PaaS non-production environments are - by default - using a sandboxed Simple Email Service Account. This means that you can only send emails to [validated recipients](https://docs.spryker.com/docs/ca/dev/email-service/verify-email-addresses.html).
```
Comment (profuel) [inline docs/ca/dev/email-service/email-deliverability.md:19]:
```suggestion
Your Spryker PaaS non-production environments are - by default - using a sandboxed Simple Email Service (or SES) Account. This means that you can only send emails to [validated recipients](https://docs.spryker.com/docs/ca/dev/email-service/verify-email-addresses.html).
```

File: docs/ca/dev/email-service/email-deliverability.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+CNAME
+NAME abcEXAMPLEHASHabc._domainkey.example.com
+VALUE abcEXAMPLEHASHabc.dkim.amazonses.com
+```  
+
+**DMARC (Domain-based Message Authentication Reporting and Conformance)**
+Purpose: Handles emails that fail SPF or DKIM checks.
+How It Works: Specifies actions (e.g., quarantine, reject) based on authentication results.
+How to configure:
+In the AWS Console, switch the the SES Dashboard. You will be able to find the DMARC DNS entries in the DMARC section under Configuration>Identities.
```
Comment (profuel) [inline docs/ca/dev/email-service/email-deliverability.md]:
```suggestion
In the AWS Console, switch to the SES Dashboard. You will be able to find the DMARC DNS entries in the DMARC section under `Configuration>Identities`.
```

File: docs/ca/dev/email-service/email-deliverability.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+You will need to set an MX and TXT record
```
Comment (profuel) [inline docs/ca/dev/email-service/email-deliverability.md:62]:
```suggestion
You will need to set an MX and TXT records:
```

**Review by feversocial — DISMISSED** [review]

File: docs/ca/dev/email-service/email-deliverability.md (hunk 4 above)
Comment (profuel) [reply docs/ca/dev/email-service/email-deliverability.md:62]: I tried to follow these instructions, and failed to find how to setup MX/TXT records.
Also, from the text below it's not clear what's the value for MX record.

Comment (profuel) [reply docs/ca/dev/email-service/email-deliverability.md:62]: https://docs.aws.amazon.com/ses/latest/dg/receiving-email-mx-record.html this page suggests that MX record is controlled in the DNS.

**Review by profuel — APPROVED** [review]

**Review by profuel — APPROVED** [review]

**Pull request merged**